.github/linters/.markdown-lint.yml

```diff
@@ -0,0 +1,18 @@
+# Default state for all rules
+default: true
+# "~MD013/line-length"
+MD013: true
```

Comment:
Wait - I think we want "MD013: false", right? I think we want to *disable* the line-length option because most of this is "one paragraph = one line". The same for the rest.

If I misunderstand this, let me know!

Reply:
I can switch it.